lib/osml/data_intake/di_dataplane.ts

```diff
@@ -246,7 +246,7 @@ export class DIDataplane extends Construct {
       buildDockerImageCode: true,
       buildFromSource: this.config.BUILD_FROM_SOURCE,
       config: {
-        CONTAINER_URI: this.config.CONTAINER_SOURCE_URI,
+        CONTAINER_URI: this.config.CONTAINER_URI,
```

Comment:
Since the keys are the same, can't we just use the spread operator here?
`config: { ...this.config }`

Reply:
The config contains ton of other configurations and unsure if we should ingest that entire config into osml_container.ts class when it only needs certain parameters. Will discuss offline